diff --git a/src/main.cpp b/src/main.cpp
index d0c900b..fa5db86 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -66,6 +66,7 @@ int main(int argc, char *argv[])
     qInstallMessageHandler(myMessageHandler);
     QLoggingCategory::setFilterRules(QStringLiteral("org.kde.*=true"));
     QQuickStyle::setStyle(QStringLiteral("Material"));
+    QIcon::setThemeName(QStringLiteral("breeze"));
 #else
     QApplication app(argc, argv);
     if (qEnvironmentVariableIsEmpty("QT_QUICK_CONTROLS_STYLE")) {
